unit_tests/test_sct_events_operator.py

```diff
@@ -66,10 +66,11 @@ def test_scylla_operator_log_event_operator_started_info(self):
 
         self.assertEqual(event, pickle.loads(pickle.dumps(event)))
         event.event_id = "9bb2980a-5940-49a7-8b08-d5c323b46aa9"
+        assert event.timestamp == 1624884782.0
         self.assertEqual(
-            "(ScyllaOperatorLogEvent Severity.NORMAL) period_type=one-time "
-            "event_id=9bb2980a-5940-49a7-8b08-d5c323b46aa9: "
-            "type=OPERATOR_STARTED_INFO regex=Starting the operator... "
-            f"line_number=0 node=N/A\n{log_record} None None: None, None",
+            '(ScyllaOperatorLogEvent Severity.NORMAL) period_type=one-time '
+            'event_id=9bb2980a-5940-49a7-8b08-d5c323b46aa9: '
+            'type=OPERATOR_STARTED_INFO regex=] operator version :'
+            ' operator/operator.go:133] operator version v1.4.0-alpha.0-34-gf13771d',
```

Comment:
Need to update it considering above comment.

Reply:
fixed